session/pingpong/hermes_channel_repository.go

```diff
@@ -252,6 +274,74 @@ func (hcr *HermesChannelRepository) handleNodeStart(payload nodeEvent.Payload) {
 	hcr.fetchKnownChannels(config.GetInt64(config.FlagChainID))
 }
 
+func (hcr *HermesChannelRepository) handleIdentityUnlock(payload identity.AppEventIdentityUnlock) {
+	hermes, err := hcr.addressProvider.GetActiveHermes(payload.ChainID)
+	if err != nil {
+		log.Err(err).Msg("failed to get active Hermes")
+		return
+	}
+	hermesChannel, err := hcr.Fetch(payload.ChainID, payload.ID, hermes)
```

Comment:
Fetch also calls BC and stuff, we only really care about local DB, so this is bit too much